### ISSUE TITLE
Future of Laravel Searchy

### ISSUE THREAD

**Issue by TomLingham** [TomLingham/Laravel-Searchy#117]
TL;DR I'm no longer maintaining this package but will help support a transition.

So, it's been a while since I've been able to give any substantial focus to this project. I understand a lot of people still use and rely on this package and I appreciate your enthusiasm for what was supposed to originally be just a hack. While I don't have the time or the desire to update and maintain this I would like to give the project back to the community as responsibly as I can.

The reasons why I am stopping maintaining Laravel Searchy are numerous, but the main reason is that I don't do any PHP development anymore. Therefore, for me to update and maintain it is a non-trivial exercise. I'm also not close enough with the community to be able to do my due diligence in understanding the direction and needs of it at large. Continuing to publish updates that I don't even test is irresponsible and I won't be doing that any more.

To facilitate this project transitioning to it's close, I will only accept pull requests that link to popular forks or alternative projects in the README (what is popular will still be up to me - likely it will require some number of GitHub stars, downloads or community participation).

**I will not be publishing any new versions.**

A big thanks to everyone who used Laravel Searchy and especially those that contributed back to the project over the years.

**Comment by necenzurat** [comment]
It would be nice of you to give make the repo public available on an organisation and let other maintain it. 
Looking for similar alternatives.

**Comment by todayqq** [comment]
https://github.com/todayqq/Laravel-Searchy

Support PHP 8.0 version
